appengine/building-an-app/update/test/server.test.js

```diff
@@ -16,15 +16,35 @@
 const path = require('path');
 const assert = require('assert');
 const utils = require('@google-cloud/nodejs-repo-tools');
+const sinon = require('sinon');
 
 const cwd = path.join(__dirname, '../');
 const requestObj = utils.getRequest({
   cwd: cwd,
   cmd: 'server',
 });
 
-beforeEach(utils.stubConsole);
-afterEach(utils.restoreConsole);
+const stubConsole = function () {
+      sinon.stub(console, `error`);
+      sinon.stub(console, `log`).callsFake((a, b) => {
+        if (
+          typeof a === `string` &&
```

Comment:
Would the tests pass without this if clause?

Reply:
Unfortunately not (at least for this very specific one). I tried with other ones and they also failed, but definitely interested in making it simpler!